docs/network-flow-visibility.md

```diff
@@ -142,26 +163,23 @@ the flow. All the IEs used by the Antrea Flow Exporter are listed below:
 
 #### Types of Flows and Associated Information
 
-Currently, the Flow Exporter feature provides visibility for Pod-to-Pod, Pod-to-Node,
-Node-to-Pod, Node-to-Node and Pod-to-Service network flows along with the associated
-statistics such as data throughput (bits per second), packet throughput (packets
-per second), cumulative byte count, cumulative packet count etc. Pod-To-Service
-flow visibility is supported only [when Antrea Proxy enabled](feature-gates.md).
+Currently, the Flow Exporter feature provides visibility for Pod-to-Pod, Pod-to-Service
+and Pod-to-External network flows along with the associated statistics such as data
+throughput (bits per second), packet throughput (packets per second), cumulative byte
+count, cumulative packet count etc. Pod-To-Service flow visibility is supported
```

Comment:
I would prefer we specify the exact set of stats that are supported if not done elsewhere; then we can remove "etc".

Reply:
done.